study-builder/studies/osteo/study.conf

```diff
@@ -2189,6 +2189,83 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    ## age-up next steps email
```

Comment:
Can you move these to the appropriate section where all the reconsent events are?